homeassistant/components/unifiled/__init__.py

```diff
@@ -1 +1,37 @@
 """Unifi LED Lights integration."""
+import logging
+
+from unifiled import unifiled
+from homeassistant.exceptions import PlatformNotReady
+from .const import KEY_API
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup(hass, config):
+    """Set up the unifiled integration."""
+    return True
```

Comment:
Don't we want to keep the possibility to set up via config yaml from the unifiled config section?

Reply:
I don't see a big reason to keep it in there. The config flow is so much more user friendly combined with that this integration has only been in the code base for 1 version i don't think it will be a huge loss.